h2o-core/src/main/java/water/util/VecUtils.java

```diff
@@ -267,6 +378,48 @@ public static Vec categoricalToInt(final Vec src) {
     return newVec;
   }
 
+  public static Vec intToDouble(final Vec src) {
+    Vec newVec = copyOver(src, Vec.T_NUM, null);
+
+    new MRTask() {
+      @Override
+      public void map(Chunk c) {
+        for (int i = 0; i < c._len; ++i) {
+          if (!c.isNA(i)) {
+            c.set(i, (c.at8(i)+EPS)*2.0-c.at8(i)-2.0*EPS);
+          }
+        }
+      }
+    }.doAll(newVec);
+    return newVec;
+  }
+
+  public static Vec categoricalToDouble(final Vec src) {
```

Comment:
This method looks the same as existing `categoricalToInt`. What is the difference?

Reply:
I forgot to change it.